index.d.ts

```diff
@@ -0,0 +1,38 @@
+interface StructInstance<StructType> {
```

Comment:
ik ben geen fan van .d.ts files die met de hand gemaakt worden of zoals bij node-homey-api gegenereerd worden vanuit de jsdocs. Ik zou in dit geval denk ik liever de tijd erin stoppen om de source naar TypeScript om te schrijven en daar de .d.ts uit laten komen. Dan weet je 100% zeker dat het klopt namelijk.

Reply:
Eens, maar er gebeurt redelijk wat magie in deze module die mogelijk niet helemaal TS compatible is. Ik kan nu niet de tijd maken om het hele project om te zetten naar TS maar zou wel graag de types gebruiken. Vind het ook goed om dit niet te mergen, dan gebruik ik de index.d.ts gewoon in mijn eigenlijk project. Maar het leek me een goed begin/misschien ook nuttig voor anderen.

Als je het eens bent om toch te mergen dan fix ik bovenstaande comments.